src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -197,20 +212,33 @@
                 //TODO: Include parameters and changelog message when implemented
                 //TODO: Add link to running report log on response, when implemented
 
+                /*
+                 * This ensures parameter values are filled before running a report
+                 * if parameter values already pass validation, it will not re-validate
+                 * Also checks if parameter has values before validating
+                 */
+                if (!this.isValidParam && this.parameterValues.length) {
```

Comment:
I think this should be avoidable with computed `isValidParam` since the run report button should never be enabled if parameter are not valid.